spec/discoverGranules/DiscoverGranulesSuccessSpec.js

```diff
@@ -6,35 +6,61 @@ const config = loadConfig();
 const lambdaStep = new LambdaStep();
 
 const taskName = 'DiscoverGranules';
-const inputTemplateFilename = './spec/discoverGranules/DiscoverGranules.input.template.json';
-const templatedInputFilename = templateFile({
-  inputTemplateFilename,
+const inputHttpTemplateFilename = './spec/discoverGranules/DiscoverGranulesHttp.input.template.json';
+const templatedHttpInputFilename = templateFile({
+  inputTemplateFilename: inputHttpTemplateFilename,
+  config: config[taskName]
+});
+const inputHttpsTemplateFilename = './spec/discoverGranules/DiscoverGranulesHttps.input.template.json';
+const templatedHttpsInputFilename = templateFile({
+  inputTemplateFilename: inputHttpsTemplateFilename,
   config: config[taskName]
 });
 
 jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
 
 describe("The Discover Granules workflow", function() {
-  let workflowExecution = null;
+  let httpWorkflowExecution = null;
+  let httpsWorkflowExecution = null;
 
   beforeAll(async function() {
-    workflowExecution = await executeWorkflow(
+    httpWorkflowExecution = await executeWorkflow(
```

Comment:
I think http and https should be separate tests, so put in a separate top level `describe` statement.